src/js/internal/primordials.js

```diff
@@ -20,22 +21,16 @@ const createSafeIterator = (factory, next) => {
 };
 
 function getGetter(cls, getter) {
-  // TODO: __lookupGetter__ is deprecated, but Object.getOwnPropertyDescriptor doesn't work on built-ins like Typed Arrays.
-  return Function.prototype.call.bind(cls.prototype.__lookupGetter__(getter));
+  var g = cls.prototype.__lookupGetter__(getter);
```

Comment:
wat

Reply:
The changes in this file are due to JSC not allowing the identifiers `call` and `apply` in a debug build. I will commit a better workaround